doc/Logging.md

```diff
@@ -0,0 +1,16 @@
+How does Happy test log work
+============================
+
+## Happy produced logs
+  * Happy host logs
+  * Happy device logs
+
+## Where to store Happy logs
+
+Both Happy host and device logs location are processed in happy/Driver.py as below:
+  * if HAPPY_LOG_DIR is found in os environment variable, happy uses HAPPY_LOG_DIR's value as log folder location to store happy test logs. For example, openweave-core tests store happy test logs along with feature folder under build folder.
```

Comment:
- Let's explain how to set HAPPY_LOG_DIR as an OS environment variable.
- Render all the code terms in code font.  So like `HAPPY_LOG_DIR`.
- Call it "Happy" (capitalized, no code font) through out the doc unless you're referring to the command itself
- "...Happy uses `HAPPY_LOG_DIR` as the log folder location..."

Reply:
done